src/http/server.cr

```diff
@@ -355,7 +355,19 @@ class HTTP::Server
     @sockets.each do |socket|
       spawn do
         until closed?
-          spawn handle_client(socket.accept? || break)
+          io = begin
+            socket.accept?
+          rescue e
+            ::puts "#{socket.class}#accept? raised but it shouldn't"
+            ::puts e
```

Comment:
`e.inspect_with_backtrace(STDOUT)`.

Reply:
`STDOUT` -> `STDERR`

Reply:
then the other line needs to be `STDERR.puts`